### ISSUE TITLE
require('himalaya') doesn't work for webpack builds in browser

### ISSUE THREAD

**Issue by robbie-c** [andrejewski/himalaya#35]
Trying to use `himalaya` in a browser, but `require('himalaya')` is returning an empty object. This is due to the code in `index.js`:

```javascript
if (typeof window === 'undefined') {
  module.exports = {default: lib, ...lib}
} else {
  window.himalaya = lib
}
```

I'm guessing this is to deal with the case where the file is being loaded from a script tag e.g. from a CDN. My advice would be to always do the `module.exports` code path, and then handle the script tag case with different build settings. If you wanted, you could build a `dist/himalaya.js` and `dist/himalaya.min.js` that people could use for this purpose.

I can make a PR if you like?

**Comment by andrejewski** [comment]
Thanks for the issue. This is addressed in v0.2.7. Please let me know if you have anymore problems.

**Comment by robbie-c** [comment]
That was fast 👍
